rails/locale/iso_639-1/en.yml

```diff
@@ -219,7 +219,7 @@ en:
     SS: South Sudan
     ST: São Tomé & Príncipe
     SV: El Salvador
-    SX: Saint Martin
+    SX: Sint Marteen
```

Comment:
This will be interesting. Three things;

- This isn't a simple rename. We're effectively changing the country we're referring to. Saint Martin and Sint Marteen are two distinct countries, one belonging to France, the other one to the Dutch Kingdom. They share a same island, but are still two distinct entities. You are right though that SX refers to Sint Marteen, and that Saint Martin already has the MF code, defined above.

- Shouldn't this be done the same way we update provinces? Shopify still has many models that reference countries by their name, and not by code, no? What will happen with all those records?

- No description, no review, nothing? Really?

<img width="806" alt="screen shot 2018-08-20 at 11 32 52 am" src="https://user-images.githubusercontent.com/596120/44332649-cdbdee80-a46c-11e8-8257-3a1a24d397ea.png">